### PR TITLE
[Wait for #2375][Unit Test] Fix unittest_interpreter

### PR DESCRIPTION
**Pull request by DonghakPark**
**Fix unittest_interpreter**

The unit test interpreter had been previously disabled due to build errors.
However, the TensorFlow Lite (TFLite) related unit tests were separated and dependencies were removed, reflecting changes made in the neural network trainer.

**Changes proposed in this PR:**
- modified:   ../test/unittest/compiler/meson.build
- modified:   ../test/unittest/compiler/unittest_interpreter.cpp

**Resolves:** 
- #2375 

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Donghak PARK <donghak.park@samsung.com>

### CODE REVIEW

File: test/unittest/compiler/meson.build [hunk 1]
```diff
@@ -3,7 +3,7 @@ test_name = 'unittest_compiler'
 test_target = [
   'compiler_test_util.cpp',
   'unittest_compiler.cpp',
-# 'unittest_interpreter.cpp',
+  'unittest_interpreter.cpp',
```
Comment (jijoongmoon) [inline test/unittest/compiler/meson.build:6]: check the indentation.

**Review by jijoongmoon — APPROVED** [review]
LGTM except for rebase.

**Comment by DonghakPark** [comment]
> LGTM except for rebase.

Rebase Done

**Review by djeong20 — APPROVED** [review]
LGTM!

**Pull request merged**